src/scss/themes/_base.scss

```diff
@@ -42,12 +42,12 @@
   --nav-svg-fill-hover: #{$secondary-text-color};
   --nav-text-color-hover: #{$secondary-text-color};
 
-  --action-button-fill-color: #{lighten($main-theme-color, 18%)};
-  --action-button-fill-color-hover: #{lighten($main-theme-color, 22%)};
-  --action-button-fill-color-active: #{lighten($main-theme-color, 5%)};
-  --action-button-fill-color-pressed: #{darken($main-theme-color, 7%)};
-  --action-button-fill-color-pressed-hover: #{darken($main-theme-color, 2%)};
-  --action-button-fill-color-pressed-active: #{darken($main-theme-color, 15%)};
+  --action-button-fill-color: #{lighten($main-theme-color, 11.5%)};
+  --action-button-fill-color-hover: #{lighten($main-theme-color, 6%)};
+  --action-button-fill-color-active: #{$main-theme-color};
+  --action-button-fill-color-pressed: #{saturate($main-theme-color, 5%)};
+  --action-button-fill-color-pressed-hover: #{darken(saturate($main-theme-color, 5%), 4%)};
+  --action-button-fill-color-pressed-active: #{darken(saturate($main-theme-color, 5%), 8%)};
```

Comment:
```suggestion
  --action-button-fill-color-pressed: #{darken(saturate($main-theme-color, 5%), 6%)};
  --action-button-fill-color-pressed-hover: #{darken(saturate($main-theme-color, 5%), 12%)};
  --action-button-fill-color-pressed-active: #{darken(saturate($main-theme-color, 5%), 15%)};
```

I'd like a _wee_ bit more contrast between unpressed and pressed here. (It makes the animation more visible, and reduces the reliance on the checkbox a bit.) On some of the light themes especially (Oaken, Majesty, Grayscale), it's really hard to tell the difference otherwise (checkbox aside).

Before (this PR):

![Screenshot from 2022-12-18 11-33-12](https://user-images.githubusercontent.com/283842/208315808-31eac3fc-e491-4b7c-bb98-9716f4d35372.png)


After (this PR + my suggestion):
![Screenshot from 2022-12-18 11-32-46](https://user-images.githubusercontent.com/283842/208315815-ff3ea21d-5bb5-4ee3-ab09-d4bacb6b1767.png)